lib/subscriber_list/hash_digest.rb

```diff
@@ -0,0 +1,27 @@
+require 'digest'
+
+class SubscriberList
```

Comment:
Since this class exists in `app/models` you would anticipate this class would live in app/models/subscriber_list`. However since this class is now free from any coupling to SubscriberList (since https://github.com/alphagov/email-alert-api/commit/331b76bcdf624cb618bc912dc1ddfd2aeb8cea95#diff-8c94b9762f86e14636fd40a443d65fbcR5) we probably don't need it coupled within namespacing.

Maybe it does make sense just in `lib/` again?